src/Providers/ValidationProvider.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+
+namespace Wepesi\App\Providers;
+
+
+use Wepesi\App\Providers\Contracts\Contracts;
+
+abstract class ValidationProvider implements Contracts
```

Comment:
Relocate provider folder to app root because it should be accessed in the entire app